src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/missingswitchdefault/InputMissingSwitchDefaultCheckSwitchExpressionsFive.java

```diff
@@ -0,0 +1,53 @@
+/*
+MissingSwitchDefault
+
+
+*/
+
+//non-compiled with javac: Compilable with Java14
```

Comment:
Attention to https://github.com/checkstyle/checkstyle/issues/14068#issue-2012342220, user reported this issue with Java 21 (there have been some changes to the switch syntax since 14). We need to add a Java 21 compilation job and compile these inputs with Java 21.

See https://github.com/checkstyle/checkstyle/pull/14025/files#diff-a74bb028b327d2ef179514069ca703e16a92ca5d0a18670c2d3daa356e032d7c and https://github.com/checkstyle/checkstyle/pull/14025/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47, you can add these in this PR.

Reply:
Added support @nrmancuso

Reply:
Please revert change in exclusion file, we cannot do that in the PR.

Reply:
> Please revert change in exclusion file, we cannot do that in the PR.

Done @nrmancuso